content/docs/components/hyperparameter.md

```diff
@@ -35,90 +101,244 @@ In this demo, 3 hyper parameters
-    Suggestion Parameters:  <nil>
+    Request Number:        3
+    Suggestion Algorithm:  random
+    Suggestion Parameters:
+      Name:   SuggestionCount
+      Value:  0
   Worker Spec:
-    Command:
-      python
-      /mxnet/example/image-classification/train_mnist.py
-      --batch-size=64
-    Image:        katib/mxnet-mnist-example
-    Worker Type:  Default
+    Go Template:
+      Raw Template:  apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.WorkerID}}
+  namespace: kubeflow
+spec:
+  template:
+    spec:
+      containers:
+      - name: {{.WorkerID}}
+        image: katib/mxnet-mnist-example
+        command:
+        - "python"
+        - "/mxnet/example/image-classification/train_mnist.py"
+        - "--batch-size=64"
+        {{- with .HyperParameters}}
+        {{- range .}}
+        - "{{.Name}}={{.Value}}"
+        {{- end}}
+        {{- end}}
+      restartPolicy: Never
 Status:
-  Best Objctive Value:          <nil>
-  Conditon:                     Running
-  Early Stopping Parameter Id:
-  Studyid:                      qb397cc06d1f8302
-  Suggestion Parameter Id:
+  Condition:                    Running
+  Early Stopping Parameter Id:  
+  Last Reconcile Time:          2019-01-18T16:30:46Z
+  Start Time:                   2019-01-18T16:30:46Z
+  Studyid:                      y456536bd1e0ad5e
+  Suggestion Count:             1
+  Suggestion Parameter Id:      i31c2adcab54f891
   Trials:
-    Trialid:  p18ee16163b85678
+    Trialid:  ka897d189e024460
     Workeridlist:
-      Objctive Value:  <nil>
-      Conditon:        Running
-      Workerid:        td08f74b9939350d
-    Trialid:           pb1be3dbe53a5cb0
+      Completion Time:  <nil>
+      Condition:        Running
+      Kind:             Job
+      Start Time:       2019-01-18T16:30:46Z
+      Workerid:         ma76ebe2b23fec02
+    Trialid:            v9ec0edbb16befd7
     Workeridlist:
-      Objctive Value:  <nil>
-      Conditon:        Running
-      Workerid:        p2b23e25cce4092c
-    Trialid:           m64209fe0867e91a
+      Completion Time:  <nil>
+      Condition:        Running
+      Kind:             Job
+      Start Time:       2019-01-18T16:30:46Z
+      Workerid:         yc5053df337dbeec
+    Trialid:            be68860be22cfce3
     Workeridlist:
-      Objctive Value:  <nil>
-      Conditon:        Running
-      Workerid:        q6258c1ac98a00a5
-Events:                <none>
+      Completion Time:  <nil>
+      Condition:        Running
+      Kind:             Job
+      Start Time:       2019-01-18T16:30:46Z
+      Workerid:         v095e6b93d87e9eb
+Events:                 <none>
 ```
 
 It should start a study and run three jobs with different parameters.
+When the `spec.Status.Condition` becomes Completed, the StudyJob is finished.
+
+### TF operator example
+
+To run tf operator example, you have to install volume for it.
+
+If you are using GKE and default StorageClass, you have to create this pvc
```

Comment:
s/pvc/persistent volume claim (PVC)

Because it's best to spell out an acronym the first time we use it, for readers who haven't seen it before.